modules/brew/brew-bash-completions.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+# shellcheck shell=sh disable=SC1091,SC2039,SC2166
+
+# Check for interactive bash and that we haven't already been sourced.
+if [ "x${BASH_VERSION-}" != x -a "x${PS1-}" != x -a "x${BREW_BASH_COMPLETION-}" = x ]; then
```

Comment:
I'd like this shellcheck error fixed https://github.com/koalaman/shellcheck/wiki/SC2268

Reply:
I think it should be like this:

`if [[ "${BASH_VERSION-}" != "" ]] && [[ "${PS1-}" != "" ]] && [[ "${BREW_BASH_COMPLETION-}" == "" ]]; then`

This bash completion is copied straight from Bluefin.
Not sure if Bluefin also copied it from somewhere, but I know this is not from official brew.

Idk why it has so many `sh`-isms tbh

Reply:
Maybe it's compatibility if someone uses POSIX mode in Bash?

Reply:
But it uses array, so it doesn't make sense that it's for `sh`, which doesn't have array functionality.

Reply:
I'm changing it back, since `bash-completions` repo uses this mechanism to combat `zsh` compatibility (which can often source from `/etc/profile.d`)